src/tracer.c

```diff
@@ -205,8 +205,10 @@ handle_syscall(pid_t pid, const struct ptrace_syscall_info *entry,
 
 	    finfo = find(pid)->finfo + fd;
 
-	    finfo->hash = hash_file(finfo->path);
-	    record_fileuse(pid, finfo->path, finfo->purpose, finfo->hash);
+	    if(finfo->path != (char *) 0) {
```

Comment:
How did you manage to have `if(` instead of `if (` (space after keyword)?
Don't you automatically invoke indent?

Reply:
@zvr I am kind of new to using git hooks and yeah it apparently doesn't work. What I've done after reading manuals was essentially modify **pre-commit** script in **.git/hooks/** so that it looks like this:

`#!/bin/bash

indent $1
exit 0`

i also removed the `.sample` as the instructions guided me to do. What am i missing?

Reply:
```bash
$ cat .git/hooks/pre-commit
#!/usr/bin/env bash

for f in $(git diff --cached --diff-filter=ACM --name-only); do
        if [[ "$f" == *.[ch] ]]; then
                INDENT_PROFILE=/home/zvr/github/gsoc2022--build-recorder/.indent.pro indent "$f" && git add "$f"
        fi
done
```

Reply:
@zvr Hmmmm git claims that the first argument is each file being commited, but i guess this approach could be used as well.

I guess what was wrong was that i wasn't `git add`ing it afterwards, and i should perhaps also add the file check.